packages/blue-migration-sdk-viem/src/positions/borrow/blue.borrow.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  type ChainId,
+  DEFAULT_SLIPPAGE_TOLERANCE,
+  type Market,
+  type MarketId,
+  MathLib,
+  type Position,
+  getChainAddresses,
+} from "@morpho-org/blue-sdk";
+import type { BlueInputBundlerOperations } from "@morpho-org/bundler-sdk-viem";
+import { maxUint256 } from "viem";
+
+export namespace MigratableBorrowPosition_Blue {
+  export interface Args {
+    marketTo: MarketId;
+    collateralAmount: bigint;
+    borrowAmount: bigint;
+    slippageFrom?: bigint;
+    slippageTo?: bigint;
+  }
+}
+
+export interface MigratableBorrowPositionConfig_Blue {
+  chainId: ChainId;
+  market: Market;
+  position: Pick<Position, "borrowShares" | "user" | "collateral">;
+}
+
+export class MigratableBorrowPosition_Blue
+  implements MigratableBorrowPositionConfig_Blue
+{
+  public readonly market: Market;
+  public readonly position: Pick<
+    Position,
+    "borrowShares" | "user" | "collateral"
+  >;
+  public readonly chainId: ChainId;
+
+  constructor(config: MigratableBorrowPositionConfig_Blue) {
+    this.market = config.market;
+    this.position = config.position;
```

Comment:
Can't we extend a `AccrualPosition` with these 2? With `supplyShares` to `0n`

Reply:
This would greatly simplify the below code with assets-shares conversion right?

Reply:
Ah but it would be incomplete because everything supply-side would be inaccurate...

Reply:
The reason we dont separate supply and borrow side positions in the SDK is because it costs the same thing: a single rpc call

Later we could still separate the 2 sides because you may want to manipulate an entity with less properties than the full position

So for now let's keep it as-is and I ref this conversation in a ticket